.github/workflows/lint.yaml

```diff
@@ -1,40 +1,10 @@
 name: Linters
 
 on:
-  push:
-    branches:
-      - main-v15
   pull_request:
 
 jobs:
-  conflicts:
```

Comment:
Doesn't hurt to run this check here

Reply:
The pre-commit job below will run the same check, so this will be redundant